packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -93,6 +93,15 @@ function isNpmInstallTask(task: vscode.Task): boolean {
   return false;
 }
 
+function isDevProxyTask(task: vscode.Task): boolean {
+  if (task) {
+    const name = task.name.trim();
+    return name === "Check dev proxy" || name === "Start dev proxy";
```

Comment:
Can this task be checked with type/command rather than name? since name may be changed by user.

Reply:
Thanks. It is updated.